Libraries/StyleSheet/LayoutPropTypes.js

```diff
@@ -20,63 +20,274 @@ var ReactPropTypes = require('ReactPropTypes');
+   *  See http://www.w3schools.com/cssref/pr_margin-right.asp
+   *  for more details.
+   */
   marginRight: ReactPropTypes.number,
+
+  /** `padding` works like `padding` in CSS.
+   *  It's like setting each of `paddingTop`, `paddingBottom`,
+   *  `paddingLeft`, and `paddingRight` to the same thing.
+   *  See http://www.w3schools.com/css/css_padding.asp
+   *  for more details.
+   */
   padding: ReactPropTypes.number,
+
+  /** Setting `paddingVertical` is like setting both of
+   *  `paddingTop` and `paddingBottom`.
+   */
   paddingVertical: ReactPropTypes.number,
+
+  /** Setting `paddingHorizontal` is like setting both of
+   *  `paddingLeft` and `paddingRight`.
+   */
   paddingHorizontal: ReactPropTypes.number,
+
+  /** `paddingTop` works like `padding-top` in CSS.
+   * See http://www.w3schools.com/cssref/pr_padding-top.asp
+   * for more details.
+   */
   paddingTop: ReactPropTypes.number,
+
+  /** `paddingBottom` works like `padding-bottom` in CSS.
+   * See http://www.w3schools.com/cssref/pr_padding-bottom.asp
+   * for more details.
+   */
   paddingBottom: ReactPropTypes.number,
+
+  /** `paddingLeft` works like `padding-left` in CSS.
+   * See http://www.w3schools.com/cssref/pr_padding-left.asp
+   * for more details.
+   */
   paddingLeft: ReactPropTypes.number,
+
+  /** `paddingRight` works like `padding-right` in CSS.
+   * See http://www.w3schools.com/cssref/pr_padding-right.asp
+   * for more details.
+   */
   paddingRight: ReactPropTypes.number,
+
+  /** `borderWidth` works like `border-width` in CSS.
+   * See http://www.w3schools.com/cssref/pr_border-width.asp
+   * for more details.
+   */
   borderWidth: ReactPropTypes.number,
+
+  /** `borderTopWidth` works like `border-top-width` in CSS.
+   * See http://www.w3schools.com/cssref/pr_border-top_width.asp
+   * for more details.
+   */
   borderTopWidth: ReactPropTypes.number,
+
+  /** `borderRightWidth` works like `border-right-width` in CSS.
+   * See http://www.w3schools.com/cssref/pr_border-right_width.asp
+   * for more details.
+   */
   borderRightWidth: ReactPropTypes.number,
+
+  /** `borderBottomWidth` works like `border-bottom-width` in CSS.
+   * See http://www.w3schools.com/cssref/pr_border-bottom_width.asp
+   * for more details.
+   */
   borderBottomWidth: ReactPropTypes.number,
+
+  /** `borderLeftWidth` works like `border-left-width` in CSS.
+   * See http://www.w3schools.com/cssref/pr_border-bottom_width.asp
+   * for more details.
+   */
   borderLeftWidth: ReactPropTypes.number,
 
+  /** Set `position` if you wish to position components using
+   *  specific numbers of logical pixels rather than using Flexbox.
+   *  You then use a combination of `top`, `left`, `right`, and
+   *  `bottom` to position the component.
+   *
+   *  It works like `position` in CSS, but in React Native you can
+   *  only set it to `absolute` or `relative`. Use `absolute` when you
+   *  want to position something relative to a parent. Use `relative`
```

Comment:
Ok, looks like you did everything right, in RN `absolute` is relative to the parent no matter what is the flow of parent